### DISCUSSION TITLE
Startup + Neotree

### DISCUSSION THREAD

**Discussion by SensorEvolve** [startup-nvim/startup.nvim#53] [Q&A] [answered]
Hi there!

When I start Nvim and the Startup screen is visible, I'm unable to use the tab key to open Neotree. Is there an easy way to fix this?

Thank you!

**Comment by max397574** [comment, marked as answer]
remove the tab mapping from startup.nvim (look at the mappings config in the readme)

**Reply by SensorEvolve** [reply]
yeah now i see. Sorry, could have figured that out. Thanks m8!
